source/jormungandr/jormungandr/scenarios/helper_classes/streetnetwork_path.py

```diff
@@ -92,6 +94,9 @@ def _direct_path_with_fp(self):
                     self._path_type,
                     self._request_id,
                 )
+            except TechnicalError as e:
+                logging.getLogger(__name__).exception('')
+                raise StreetNetworkException(response_pb2.Error.service_unavailable, e.data["message"])
             except Exception as e:
                 logging.getLogger(__name__).exception('')
                 return None
```

Comment:
(I know that's not your code) we should raise en exception here, if we follow the logic analogously...